packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -118,11 +119,12 @@ class ColdRunner extends ResidentRunner {
       final FlutterDevice device = flutterDevices.first;
       if (device.vmService != null) {
         globals.printStatus('Tracing startup on ${device.device.name}.');
+        final String outputPath = globals.platform.environment[kFlutterTestOutputsDirEnvName] ?? getBuildDirectory();
```

Comment:
Should this ever be used outside `drive`?

Reply:
Meaning, should `flutter run --trace-startup` save to that location? Or should it only be used for `flutter drive` output since the only docs I can find are for `flutter_driver` https://api.flutter.dev/flutter/flutter_driver/testOutputsDirectory.html